.changes/v3.1.0/594-features.md

```diff
@@ -0,0 +1,2 @@
+**New Data source:** `vcd_resource_info` info about VCD entities [GH-594]
```

Comment:
I believe a better name is `vcd_resource_list` and it looks you are even using it :) The old name is here and in PR name, description and maybe somewhere else too.

Reply:
fixed